deploy/ft/ft-create-sdk.js

```diff
@@ -0,0 +1,49 @@
+//in this script, we check if the funder owns the NFTs they are adding to the contract. If they don't, throw an error
+const { FUNDING_ACCOUNT_ID, FUNDER_INFO, NETWORK_ID, NUM_KEYS, DROP_METADATA, DEPOSIT_PER_USE, DROP_CONFIG, KEYPOM_CONTRACT, NFT_DATA, NFT_CONTRACT_ID, NFT_METADATA} = require("./configurations");
+
+const path = require("path");
+const homedir = require("os").homedir();
+const { writeFile, mkdir, readFile } = require('fs/promises');
+const { initKeypom, createDrop, getDrops } = require("keypom-js");
+
+//funder is account to sign txns, defaults to benjiman.testnet
+//numKeys default is 10, deposit defalt is 10, default drop config 1 use per key and delete on empty, metadata empty, funderBalance false
```

Comment:
Let's keep comment styles consistent and have a standard for how we do comments: try to have a space after each comment and start each sentence with a capital letter (unless it's a variable)